packages/sandbox/src/components/FieldPluginSandbox.tsx

```diff
@@ -526,6 +543,14 @@ export const FieldPluginSandbox: FunctionComponent = () => {
           />
         </AccordionDetails>
       </Accordion>
+      <Accordion defaultExpanded>
+        <AccordionSummary>
+          <Typography variant="h3">Data</Typography>
```

Comment:
Since Data is already used for another type of content, I will rename this as:

```suggestion
          <Typography variant="h3">Editor Preview</Typography>
```